code/obj/item/organs/stomach.dm

```diff
@@ -55,6 +55,11 @@
 	proc/consume(atom/movable/AM)
 		if (AM in src.stomach_contents)
 			return
+		if (istype(AM, /obj/item/organ)) // organs don't have a digestion proc, so just delete them instead
```

Comment:
Why not implement a digestion thing that does nothing/deletes them?

Reply:
Organs being immediately deleted prevents people from vomiting up the organs before the digestion loop and re-consuming them, therefore preventing people from getting unlimited amounts of metachroitic factor (saurian points system) from the same organ. Also, most organs dont hold reagents, and hearts (the only organs to my knowledge that do hold reagents) immediately transfer them all to the consumer when eaten anyways, making a digestion proc unnecessary.

Reply:
Updated it to use a proc that deletes organs in the stomach every life cycle, seems to work

Reply:
even if the performance might be wack this is a lot cleaner thank you